src/main/kotlin/no/nav/syfo/infrastructure/kafka/ExpiredForhandsvarselProducer.kt

```diff
@@ -0,0 +1,33 @@
+package no.nav.syfo.infrastructure.kafka
+
+import no.nav.syfo.application.IExpiredForhandsvarslerProducer
+import no.nav.syfo.domain.PersonIdent
+import no.nav.syfo.domain.Varsel
+import org.apache.kafka.clients.producer.KafkaProducer
+import org.apache.kafka.clients.producer.ProducerRecord
+import org.slf4j.LoggerFactory
+import java.util.*
+
+class ExpiredForhandsvarselProducer(private val producer: KafkaProducer<String, String>) :
+    IExpiredForhandsvarslerProducer {
+
+    override fun send(personIdent: PersonIdent, varsel: Varsel): Result<Varsel> =
+        try {
+            producer.send(
+                ProducerRecord(
+                    TOPIC,
+                    UUID.randomUUID().toString(),
+                    "",
+                )
+            ).get()
+            Result.success(varsel)
+        } catch (e: Exception) {
+            log.error("Exception was thrown when attempting to send forhandsvarsel to syfooversiktsrv: ${e.message}")
```

Comment:
Tenker det holder med noe ala `"... attempting to send expired forhandsvarsel"`. Vi produserer bare til en kafka-topic som vi må definere i denne appen. Hvem som konsumerer bør/trenger ikke koden her ha noe forhold til.